linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_privileged_commands_init/rule.yml

```diff
@@ -27,9 +27,11 @@ references:
     nist: AU-12(c)
     srg: SRG-OS-000477-GPOS-00222
 
-{{{ ocil_fix_srg_privileged_command("init", "/usr/sbin/") }}}
```

Comment:
The removal of `/usr/sbin` from this place doesn't seem to be right because the default value of this parameter is `/usr/bin`, that means that the generated OCIL and fixtext in RHEL 9 (and probably other products as well) contain an incorrect path. Can you take a look into this problem and make sure that you don't break RHEL 9 or other products?

Reply:
I have added a Jinja macro, that sets the path for the rule, based on the os. I hope it is a viable solution.